src/sidebar/store/modules/annotations.ts

```diff
@@ -603,6 +603,31 @@ const usersWhoAnnotated = createSelector(
   },
 );
 
+/**
+ * Return the list of unique users who have been mentioned in an annotation.
+ */
+const usersWhoWereMentioned = createSelector(
+  (state: State) => state.annotations,
+  annotations => {
+    const usersMap = new Map<
+      string,
+      { userid: string; username: string; displayName: string | null }
+    >();
```

Comment:
This is fine, although you could also just keep a `Set<string>` of unique user IDs and an array of all the unique `UserItem`s, as opposed to building a map and turning the values back into an array at the end.

Reply:
I used this approach because it allows to filter out duplicates and also map to the expected structure in just one iteration.

Reply:
Actually, the `[...usersMap.values()]` structure coming after involves a second iteration 😅

Reply:
Since two iterations were happening anyway, I changed it to `mentions.filter(...).map(...)`, where the filtering is done based on a `Set<string>`.